config/mbrave.yml

```diff
@@ -636,6 +636,323 @@ development: &1
       - PB1R_bc1190_rc
       - PB1R_bc1191_rc
       - PB1R_bc1192_rc
+  Bioscan_forward_96_v3:
```

Comment:
Put v3 config at the top of the file? At the moment this sequence is v2, then v1, then v3